### DISCUSSION TITLE
How to convert to YUV color space?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#1084] [Q&A] [answered]
Hi,

How could we convert sRGB to YUV color space?

Thanks

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

Have you looked at the `colour.RGB_to_YCbCr` definition? Or its sister one `colour.matrix_YCbCr`?

Cheers,

Thomas

**Reply by hminle** [reply]
Oh, thank @KelSolaar . So we can use YCrCr and YUV interchangeably? I thought they are different. Or maybe we are dealing with digital (not analog) images, then they are the same?

**Reply by thomasmansencal** [reply]
There are many variants of YUV, it is generally a class of space that separates luma from chroma and applies sub-sampling on the chroma components. Provided you encode with the set of weights and decode with the same, things should be all good.

**Reply by hminle** [reply]
Thank @KelSolaar .

Another question about this function `colour.RGB_to_YCbCr`. When applying this function to RGB values, what is that "RGB"?
Is it linearized sRGB? (I mean the sRGB values after we de-gamma)

**Reply by thomasmansencal** [reply]
Great question and no! I used "luma" specifically, implying that it is on non-linear data.
